README.rst

```diff
@@ -317,6 +317,20 @@ Load kernel modules and add them to `/etc/modules`:
             - tp_smapi
             - 8021q
 
+Configure kernel modules with additional options to `/etc/modprobe.d`
+following example will add `/etc/modprobe.d/nf_conntrack.conf` file
+with line `options nf_conntrack hashsize=262144`:
+
+.. code-block:: yaml
+
+    linux:
+      system:
+        kernel:
+          kmod_config:
```

Comment:
Rename it to only `module` and change structure slightly to allow further extending:

```
linux:
  system:
    kernel:
      module:
        nf_conntrack:
          enabled: true  # to allow enabling/disabling (blacklisting) module
          option: # It's better to use dicts than lists where there's always key
            hashsize: 262144
```
Then you'll use something like `options {{ name }} {% for k, v in module.option.iteritems() %}{{ k }}={{ v }}{% endfor %}`